src/front/wgsl/parse/mod.rs

```diff
@@ -2213,7 +2213,7 @@ impl Parser {
                 let members = self.struct_body(lexer, ctx)?;
                 Some(ast::GlobalDeclKind::Struct(ast::Struct { name, members }))
             }
-            (Token::Word("type"), _) => {
+            (Token::Word("type"), _) | (Token::Word("alias"), _) => {
```

Comment:
I think it would be best to simply remove support for `type` altogether. Everyone will need to make that change soon eventually.

Reply:
Agreed!